roles/lib_openshift/src/class/oc_objectvalidator.py

```diff
@@ -72,6 +72,15 @@ def _is_invalid_namespace(namespace):
                 all_invalid[invalid_msg] = invalid
 
         if failed:
-            return {'failed': True, 'msg': 'All objects are not valid.', 'state': 'list', 'results': all_invalid}
+            return {
+                'failed': True,
+                'msg': (
+                    "All objects are not valid.  If you are a supported customer please contact "
+                    "Red Hat Support providing the complete output above. If you are not a customer "
+                    "please contact users@lists.openshift.redhat.com for assistance."
+                    ),
```

Comment:
I'm not sure if this ending ```()``` location will cause a ``flake8`` or ``pylint`` error. But the change looks good.